### PR TITLE
Postbin

### PR DESCRIPTION
**Pull request by tiseysoft**

### CODE REVIEW

**Comment by ppolinaua** [comment]
@tiseysoft Refer to [Postbin credentials]- a broken link

**Comment by tiseysoft** [comment]
> @tiseysoft Refer to [Postbin credentials]- a broken link

I cannot find a link for postbin, is the page not available?

**Comment by ppolinaua** [comment]
@StarfallProjects do we have a credentials page for postbin?

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.postbin.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Postbin
 
-[Postbin](https://www.toptal.com/developers/postbin/){:target=_blank .external-link} is a service that collects all requests to a URL, which you can use to test your API Clients or your WebHooks.
+The Postbin node allows you to automate work in Postbin, and integrate Postbin with other applications. n8n has built-in support for a wide range of Postbin features, including collecting URL requests. 
+
+On this page, you'll find a list of operations the Postbin node supports and links to more resources.
+
+!!! note "Credentials"
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.postbin.md:7]:
```suggestion
```

**Pull request merged**